docs/github-connector/README.md

```diff
@@ -0,0 +1,15 @@
+# GitHub Connector
+
+## Overview
+
+The GitHub Connector is an additional tool for handling GitHub events in Kyma. It registers GitHub API and events in Kyma Application Registry. Then, it converts events incoming from connected GitHub webhooks into format acceptable by Kyma Event Bus and forwards them. For now it handles those events:
```

Comment:
```suggestion
The GitHub Connector is an additional tool for handling GitHub events in Kyma. It registers GitHub API and events in Kyma Application Registry. Then, it converts events incoming from connected GitHub webhooks to format acceptable by Kyma Event Bus and forwards them. For now it handles these events:
```

- `The GitHub Connector is an additional tool` - Additional to what? (Also, do we use `the` in front of GitHub connector? Just checking to make sure.)
- When you convert A **into** B, A becomes B. See [this](https://www.usingenglish.com/forum/threads/99446-convert-into-convert-to). 
- `to the format acceptable`, or `to a format acceptable`, if there are more acceptable formats
- @`For now`: As per [guidelines](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/04-formatting.md):
  > **NOTE:** Do not use words such as "currently" or "now" to indicate that something is in the transitional phase of development. Avoid promising anything and mention only those components and functionalities that are already in use.

  You can rephrase it as: `The GitHub Connector handles:` or `The GitHub connector handles these events:`.

- `and forwards them` where?

Reply:
I'm also not sure about that `the`. I've seen that Kyma components have it in front of their names (like the Application Connector).